### ISSUE TITLE
Release: release notes and changelog

### ISSUE THREAD

**Issue by dexX7** [zathras-crypto/omnicore#52]
Ideally there would be some release notes and a change log similar to:

https://github.com/bitcoin/bitcoin/blob/0.9/doc/release-notes.md

**Comment by zathras-crypto** [comment]
I started to try and work this, but it's an absolute PITA because we don't have a previous state to work from.  I don't have a clear path to do this, I tried walking commits and doing a changelog like that but it seems entirely fruitless (eg a changelog that has "X added" then a few lines down "X removed" makes no sense because in the end X is unchanged). 

Trying to think about how to _start_ a changelog half way through a project where one hasn't existed before - open to suggestions on process for generating this documentation - anyone with experience of doing this type of thing want to weigh in here?

Basically the result I think we're trying to aim for is what has changed between this version and 0.0.9-rel and currently the only way I can think to do this is to step through all the work we've done and try and work it all out manually via diffing the 0.0.9.1 code against 0.0.9 code instead of using commit history, but that seems like an awfully long winded process :(
